diff -ur wxPython-src-2.8.9.1/wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp	2008-09-28 15:11:10.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp	2008-12-27 16:35:37.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/contrib/glcanvas/gtk/glcanvas_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/contrib/glcanvas/gtk/glcanvas_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/contrib/glcanvas/gtk/glcanvas_wrap.cpp	2008-09-28 15:11:01.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/contrib/glcanvas/gtk/glcanvas_wrap.cpp	2008-12-27 16:35:37.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/contrib/stc/gtk/stc_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/contrib/stc/gtk/stc_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/contrib/stc/gtk/stc_wrap.cpp	2008-09-28 15:11:05.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/contrib/stc/gtk/stc_wrap.cpp	2008-12-27 16:35:37.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/animate_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/animate_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/animate_wrap.cpp	2008-09-28 15:10:59.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/animate_wrap.cpp	2008-12-27 14:26:51.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/aui_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/aui_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/aui_wrap.cpp	2008-09-28 15:10:58.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/aui_wrap.cpp	2008-12-27 14:26:51.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/calendar_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/calendar_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/calendar_wrap.cpp	2008-09-28 15:10:31.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/calendar_wrap.cpp	2008-12-27 14:26:51.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/combo_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/combo_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/combo_wrap.cpp	2008-09-28 15:10:34.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/combo_wrap.cpp	2008-12-27 14:26:51.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/_controls_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_controls_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/_controls_wrap.cpp	2008-09-28 15:10:25.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_controls_wrap.cpp	2008-12-27 14:26:52.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/_core_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_core_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/_core_wrap.cpp	2008-09-28 15:10:12.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_core_wrap.cpp	2008-12-27 14:26:52.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/_gdi_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_gdi_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/_gdi_wrap.cpp	2008-09-28 15:10:16.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_gdi_wrap.cpp	2008-12-27 14:26:53.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/grid_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/grid_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/grid_wrap.cpp	2008-09-28 15:10:38.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/grid_wrap.cpp	2008-12-27 14:26:53.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/html_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/html_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/html_wrap.cpp	2008-09-28 15:10:41.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/html_wrap.cpp	2008-12-27 14:26:53.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/media_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/media_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/media_wrap.cpp	2008-09-28 15:10:43.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/media_wrap.cpp	2008-12-27 14:26:53.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/_misc_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_misc_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/_misc_wrap.cpp	2008-09-28 15:10:29.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_misc_wrap.cpp	2008-12-27 14:26:54.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/richtext_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/richtext_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/richtext_wrap.cpp	2008-09-28 15:10:53.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/richtext_wrap.cpp	2008-12-27 14:26:54.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/webkit_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/webkit_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/webkit_wrap.cpp	2008-09-28 15:10:44.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/webkit_wrap.cpp	2008-12-27 14:26:54.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/_windows_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_windows_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/_windows_wrap.cpp	2008-09-28 15:10:20.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/_windows_wrap.cpp	2008-12-27 14:26:55.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/wizard_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/wizard_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/wizard_wrap.cpp	2008-09-28 15:10:47.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/wizard_wrap.cpp	2008-12-27 14:26:55.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
diff -ur wxPython-src-2.8.9.1/wxPython/src/gtk/xrc_wrap.cpp wxPython-src-2.8.9.1-literal/wxPython/src/gtk/xrc_wrap.cpp
--- wxPython-src-2.8.9.1/wxPython/src/gtk/xrc_wrap.cpp	2008-09-28 15:10:49.000000000 -0700
+++ wxPython-src-2.8.9.1-literal/wxPython/src/gtk/xrc_wrap.cpp	2008-12-27 14:26:55.000000000 -0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
