--- wxPython/src/gtk/animate_wrap.cpp.str	2014-02-17 15:07:44.315418401 +0800
+++ wxPython/src/gtk/animate_wrap.cpp	2014-02-17 15:07:44.335418956 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/aui_wrap.cpp.str	2014-02-17 15:07:44.345419236 +0800
+++ wxPython/src/gtk/aui_wrap.cpp	2014-02-17 15:07:44.445422030 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/calendar_wrap.cpp.str	2014-02-17 15:07:44.445422030 +0800
+++ wxPython/src/gtk/calendar_wrap.cpp	2014-02-17 15:07:44.475422868 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/combo_wrap.cpp.str	2014-02-17 15:07:44.475422868 +0800
+++ wxPython/src/gtk/combo_wrap.cpp	2014-02-17 15:07:44.515423986 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/_controls_wrap.cpp.str	2014-02-17 15:07:44.515423986 +0800
+++ wxPython/src/gtk/_controls_wrap.cpp	2014-02-17 15:07:44.715429572 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/_core_wrap.cpp.str	2014-02-17 15:07:44.725429852 +0800
+++ wxPython/src/gtk/_core_wrap.cpp	2014-02-17 15:07:44.985437114 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/dataview_wrap.cpp.str	2014-02-17 15:07:44.995437394 +0800
+++ wxPython/src/gtk/dataview_wrap.cpp	2014-02-17 15:07:45.065439350 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/_gdi_wrap.cpp.str	2014-02-17 15:07:45.065439350 +0800
+++ wxPython/src/gtk/_gdi_wrap.cpp	2014-02-17 15:07:45.225443820 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/glcanvas_wrap.cpp.str	2014-02-17 15:07:45.225443820 +0800
+++ wxPython/src/gtk/glcanvas_wrap.cpp	2014-02-17 15:07:45.245444378 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/grid_wrap.cpp.str	2014-02-17 15:07:45.245444378 +0800
+++ wxPython/src/gtk/grid_wrap.cpp	2014-02-17 15:07:45.335446892 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/html2_wrap.cpp.str	2014-02-17 15:07:45.335446892 +0800
+++ wxPython/src/gtk/html2_wrap.cpp	2014-02-17 15:07:45.365447730 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/html_wrap.cpp.str	2014-02-17 15:07:45.365447730 +0800
+++ wxPython/src/gtk/html_wrap.cpp	2014-02-17 15:07:45.435449686 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/media_wrap.cpp.str	2014-02-17 15:07:45.435449686 +0800
+++ wxPython/src/gtk/media_wrap.cpp	2014-02-17 15:07:45.455450244 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/_misc_wrap.cpp.str	2014-02-17 15:07:45.455450244 +0800
+++ wxPython/src/gtk/_misc_wrap.cpp	2014-02-17 15:07:45.605454434 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/propgrid_wrap.cpp.str	2014-02-17 15:07:45.605454434 +0800
+++ wxPython/src/gtk/propgrid_wrap.cpp	2014-02-17 15:07:45.775459184 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/richtext_wrap.cpp.str	2014-02-17 15:07:45.775459184 +0800
+++ wxPython/src/gtk/richtext_wrap.cpp	2014-02-17 15:07:45.905462814 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/stc_wrap.cpp.str	2014-02-17 15:07:45.915463094 +0800
+++ wxPython/src/gtk/stc_wrap.cpp	2014-02-17 15:07:46.005465608 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/webkit_wrap.cpp.str	2014-02-17 15:07:46.015465888 +0800
+++ wxPython/src/gtk/webkit_wrap.cpp	2014-02-17 15:07:46.035466446 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/_windows_wrap.cpp.str	2014-02-17 15:07:46.035466446 +0800
+++ wxPython/src/gtk/_windows_wrap.cpp	2014-02-17 15:07:46.185470636 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/wizard_wrap.cpp.str	2014-02-17 15:07:46.185470636 +0800
+++ wxPython/src/gtk/wizard_wrap.cpp	2014-02-17 15:07:46.215471474 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/src/gtk/xrc_wrap.cpp.str	2014-02-17 15:07:46.215471474 +0800
+++ wxPython/src/gtk/xrc_wrap.cpp	2014-02-17 15:07:46.255472592 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp~	2013-12-16 21:52:12.000000000 +0800
+++ wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp	2014-02-17 15:19:06.354656816 +0800
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
