--- wxPython/setup.py.link	2011-05-07 08:26:12.802491052 +0200
+++ wxPython/setup.py	2011-05-07 08:31:45.702709273 +0200
@@ -179,7 +179,7 @@
                 define_macros = defines,
 
                 library_dirs = libdirs,
-                libraries = libs,
+                libraries = libs + ['X11', 'gdk-x11-2.0', 'gobject-2.0'],
 
                 extra_compile_args = cflags,
                 extra_link_args = lflags,
